assets/i18n/zh-tw.json

```
@@ -0,0 +1,269 @@
+{
+    "@metadata": {
+        "authors": [
+            "tw-dog",
+            "getlocalization contributors",
+            "etc"
+        ],
+        "last-updated": "2021-05-21",
+        "locale": "zh-tw",
+        "message-documentation": "qqq",
+        "website-url": "dogecoin.com"
+    },
+    "dgc-title": "狗狗幣",
+    "dgc-title-firstcap": "狗狗幣", 
+    "dgc-title-caps": "狗狗幣", 
+    "dgc-alt-watch-video": "看影片", 
+    "dgc-alt-logo": "狗狗幣 圖標", 
+    "dgc-header-desc": "狗狗幣是一種開源的加密貨幣，全世界的狗狗都很喜歡。", 
+    "dgc-start-using": "開始使用狗狗幣吧：",
+    "dgc-wallet-windows": " Windows", 
+    "dgc-wallet-osx": " macOS", 
+    "dgc-wallet-linux": " Linux", 
+    "dgc-wallet-ios": " iOS", 
+    "dgc-wallet-android": " Android", 
+    "dgc-wallet-blackberry": " Blackberry", 
+    "dgc-wallet-browser": " 網頁版", 
+    "dgc-wallet-source": " 原始碼", 
+    "dgc-scroll-button-link": "點擊後向下滾動", 
+    "dgc-scroll-button-alt": "向下滾動按鈕", 
+    "dgc-dogepng-alt": "狗", 
+    "dgc-what-is-doge": "什麼是狗狗幣？", 
+    "dgc-quick-desc": "狗狗幣是去中心化加密貨幣，讓您能輕鬆的在網路上傳送貨幣。簡而言之就是 \"網路貨幣\"",
+    "dgc-get-started-now": "開始使用狗狗幣", 
+    "dgc-learn-more": "了解更多", 
+    "dgc-what-with-dog": "什麼是狗狗幣 它是狗嗎？", 
+    "dgc-doge-explained": "\"狗\" 只是個有趣的命名, 就像是吉祥物！作為流行迷因的日本柴犬就永遠鑲嵌進狗狗幣中了！", 
```

```suggestion
    "dgc-doge-explained": "「狗」只是個有趣的命名, 就像是吉祥物！作為流行迷因的日本柴犬就永遠鑲嵌進狗狗幣中了！", 
```
I think we need rephrasing on this one since it doesn't sound connected in Chinese. I know it's hard since this sentence should have a much different linguistic structure in Chinese from that of English.

Original: "Doge" is our fun, friendly mascot! The Shiba Inu is a Japanese breed of dog that was popularized as an online meme and represents Dogecoin.

---

ok, let me think.

---

Keep doge untranslated would be fine since "doge" never being a meme in Taiwan, IMO.